{
	"id": "172726",
	"key": "AC-6056",
	"fields": {
		"issuetype": {
			"id": "1",
			"description": "A problem which impairs or prevents the functions of the product.",
			"name": "Bug",
			"subtask": false
		},
		"project": {
			"id": "12217",
			"key": "AC",
			"name": "Appcelerator - INBOX",
			"projectCategory": {
				"id": "10000",
				"description": "",
				"name": "Customer Service"
			}
		},
		"resolution": {
			"id": "3",
			"description": "The problem is a duplicate of an existing issue.",
			"name": "Duplicate"
		},
		"resolutiondate": "2018-12-20T21:59:29.000+0000",
		"created": "2018-12-11T08:21:42.000+0000",
		"labels": [],
		"versions": [],
		"issuelinks": [],
		"assignee": {
			"name": "shossain",
			"key": "shossain",
			"displayName": "Shak Hossain",
			"active": false,
			"timeZone": "America/Los_Angeles"
		},
		"updated": "2018-12-20T21:59:30.000+0000",
		"status": {
			"description": "A resolution has been taken, and it is awaiting verification by reporter. From here issues are either reopened, or are closed.",
			"name": "Resolved",
			"id": "5",
			"statusCategory": {
				"id": 3,
				"key": "done",
				"colorName": "green",
				"name": "Done"
			}
		},
		"components": [
			{
				"id": "14551",
				"name": "Studio",
				"description": "Please enter tickets related to Titanium Studio here."
			},
			{
				"id": "14548",
				"name": "Titanium SDK & CLI",
				"description": "Please enter tickets related to the MobileSDK here."
			}
		],
		"description": "I get below error\r\n\r\n\r\n\r\n{code:java}\r\n[WARN] :   TiVerify: (Timer-0) [5009,5009] Verifying module licenses...\r\n[ERROR] :  TiApplication: (Timer-0) [656,5665] Sending event: exception on thread: Timer-0 msg:java.lang.IncompatibleClassChangeError: Class 'ti.modules.titanium.network.NonValidatingSSLSocketFactory' does not implement interface 'org.apache.http.conn.scheme.SocketFactory' in call to 'java.net.Socket org.apache.http.conn.scheme.SocketFactory.createSocket()' (declaration of 'org.apache.http.impl.conn.DefaultClientConnectionOperator' appears in /system/framework/org.apache.http.legacy.boot.jar); Titanium 7.1.0,2018/03/14 13:40,undefined\r\n[ERROR] :  TiApplication: java.lang.IncompatibleClassChangeError: Class 'ti.modules.titanium.network.NonValidatingSSLSocketFactory' does not implement interface 'org.apache.http.conn.scheme.SocketFactory' in call to 'java.net.Socket org.apache.http.conn.scheme.SocketFactory.createSocket()' (declaration of 'org.apache.http.impl.conn.DefaultClientConnectionOperator' appears in /system/framework/org.apache.http.legacy.boot.jar)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:145)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:169)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:124)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:366)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:560)\r\n[ERROR] :  TiApplication: \tat org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:514)\r\n[ERROR] :  TiApplication: \tat org.appcelerator.titanium.TiVerify.run(Unknown Source:280)\r\n[ERROR] :  TiApplication: \tat java.util.TimerThread.mainLoop(Timer.java:555)\r\n[ERROR] :  TiApplication: \tat java.util.TimerThread.run(Timer.java:505)\r\n{code}\r\n",
		"attachment": [],
		"flagged": false,
		"summary": "Ti.Module verify crash upon install thru Appcelerator",
		"creator": {
			"name": "dyan02",
			"key": "dyan02",
			"displayName": "ardy wongso",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"subtasks": [],
		"reporter": {
			"name": "dyan02",
			"key": "dyan02",
			"displayName": "ardy wongso",
			"active": true,
			"timeZone": "America/Los_Angeles"
		},
		"environment": "using SDK 7.1.0",
		"comment": {
			"comments": [
				{
					"id": "444456",
					"author": {
						"name": "dyan02",
						"key": "dyan02",
						"displayName": "ardy wongso",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "I try only add FB module, results still the same",
					"updateAuthor": {
						"name": "dyan02",
						"key": "dyan02",
						"displayName": "ardy wongso",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2018-12-11T08:22:08.000+0000",
					"updated": "2018-12-11T08:22:08.000+0000"
				},
				{
					"id": "444469",
					"author": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Hello,\r\n\r\nThanks for raising this. This is a known issue. Our engineering team is working to fix this as soon as possible. \r\n\r\n\r\nThanks",
					"updateAuthor": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2018-12-11T11:37:52.000+0000",
					"updated": "2018-12-11T11:37:52.000+0000"
				},
				{
					"id": "444579",
					"author": {
						"name": "aibanez@kpitec.com",
						"key": "aibanez@kpitec.com",
						"displayName": "aibanez",
						"active": true,
						"timeZone": "America/Santiago"
					},
					"body": "I'm having the same problem, but I don't know how when I haven't made any changes. I have an app made with Titanium SDK 6.0.3 and it suddenly started shutting down. Through logcat I saw that the crash was logged with the same one from this issue. I hope this is not fixed with new Titanium SDK version 'cause I can't upgrade given that, by my understanding, I can't make apps compatibles with Android 5.0 using Titanium SDK with higher version that the one I'm using. Help please.",
					"updateAuthor": {
						"name": "aibanez@kpitec.com",
						"key": "aibanez@kpitec.com",
						"displayName": "aibanez",
						"active": true,
						"timeZone": "America/Santiago"
					},
					"created": "2018-12-13T13:32:11.000+0000",
					"updated": "2018-12-13T13:32:11.000+0000"
				},
				{
					"id": "444637",
					"author": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "[~dyan02],\r\n\r\nThanks for your feedback. Can you please check on your end and let us know your current status of the issue.",
					"updateAuthor": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2018-12-17T04:38:13.000+0000",
					"updated": "2018-12-17T04:38:13.000+0000"
				},
				{
					"id": "444659",
					"author": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"body": "Hello [~dyan02], The issue is fixed on SDK 7.4.2 Version. You have to rebuild the app with the sdk and provide an update to your app. You will lose Android 5 support though. Thanks.",
					"updateAuthor": {
						"name": "sdarda",
						"key": "sdarda",
						"displayName": "Sharif AbuDarda",
						"active": false,
						"timeZone": "Asia/Dhaka"
					},
					"created": "2018-12-17T21:34:04.000+0000",
					"updated": "2018-12-17T21:34:04.000+0000"
				},
				{
					"id": "444671",
					"author": {
						"name": "dyan02",
						"key": "dyan02",
						"displayName": "ardy wongso",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"body": "Hi yes it works by now, but what do you mean when \"lose android 5 support\"? means android with os lollipop and below unable to install the app?",
					"updateAuthor": {
						"name": "dyan02",
						"key": "dyan02",
						"displayName": "ardy wongso",
						"active": true,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2018-12-18T02:51:27.000+0000",
					"updated": "2018-12-18T02:51:27.000+0000"
				},
				{
					"id": "444674",
					"author": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"body": "[~dyan02],\r\n\r\nYes. Glad to hear that the issue is resolved now.",
					"updateAuthor": {
						"name": "rmitro",
						"key": "rmitro",
						"displayName": "Rakhi Mitro",
						"active": false,
						"timeZone": "America/Los_Angeles"
					},
					"created": "2018-12-18T04:01:59.000+0000",
					"updated": "2018-12-18T04:01:59.000+0000"
				}
			],
			"maxResults": 8,
			"total": 8,
			"startAt": 0
		}
	}
}